codegenerator/cli/src/rescript_types.rs

```diff
@@ -424,14 +429,16 @@ impl RescriptTypeIdent {
         }
     }
 
-    pub fn to_rescript_schema(&self, with_db_coerce: bool) -> String {
+    pub fn to_rescript_schema(&self, mode: &RescriptSchemaMode) -> String {
```

Comment:
Why does it need to be a reference out of interest? Can't the fn just take ownership of the flag?

Reply:
It would require to do .clone in a few places, so I thought it's simpler to work with a reference.